### PR TITLE
x11-wm/selectwm: EAPI6, use HTTPS, update patch

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This my take on updating this package towards EAPI6.

Please review.

### CODE REVIEW

File: x11-wm/selectwm/selectwm-0.4.1.ebuild [hunk 1]
```diff
@@ -1,37 +1,35 @@
...
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
+	eapply "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
 		"${FILESDIR}"/${P}-glibc-2.10.patch \
 		"${FILESDIR}"/${P}-nostrip.patch
 	eautoreconf
+	default
```
Comment (Moiman) [inline x11-wm/selectwm/selectwm-0.4.1.ebuild]: move default before eautoreconf

**Comment by mm1ke** [comment]
rebased, any chance we can merge this? :)

File: x11-wm/selectwm/selectwm-0.4.1.ebuild [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
+DOCS=( AUTHORS README sample.xinitrc )
+
 RDEPEND="x11-libs/gtk+:2
 	dev-libs/glib:2"
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
+	eapply "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
```
Comment (Moiman) [inline x11-wm/selectwm/selectwm-0.4.1.ebuild]: Can you use PATCHES=() ?

Comment (mm1ke) [reply x11-wm/selectwm/selectwm-0.4.1.ebuild]: Hi, i've updated the PR to use the PATCHES Variable

File: x11-wm/selectwm/selectwm-0.4.1.ebuild [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
+EAPI=6
+inherit autotools
 
 DESCRIPTION="window manager selector tool"
-HOMEPAGE="http://ordiluc.net/selectwm"
-SRC_URI="http://ordiluc.net/selectwm/${P}.tar.bz2"
+HOMEPAGE="https://ordiluc.net/selectwm"
+SRC_URI="https://ordiluc.net/selectwm/${P}.tar.bz2"
 
 LICENSE="GPL-2"
```
Comment (Moiman) [inline x11-wm/selectwm/selectwm-0.4.1.ebuild]: License is gpl-2+

Comment (mm1ke) [reply x11-wm/selectwm/selectwm-0.4.1.ebuild]: thanks for pointing that out, fixed it :)

**Comment by flappyports** [comment]
@mm1ke Please make this a revbump as @mgorny asked and I will merge it.

**Comment by mm1ke** [comment]
@flappyports @mgorny I've added a r1 with all the changes and only fixed the https for the original file. I've also dropped the stable keywords for the r1 and fixed all repoman warnings.
Please review.

**Comment by mm1ke** [comment]
Sorry, i've accidentally removed this branch before ;)

**Pull request closed without merging**